broker/staticconfiguration/impl.go

```diff
@@ -0,0 +1,46 @@
+package staticconfiguration
+
+import (
+	"errors"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+)
+
+const defaultServicePort = 443
+const defaultStatusPort = 6930
```

Comment:
Please put `defaultStatusPort` into a constants package so that other code can import it.